src/main/java/slimeknights/tconstruct/library/MaterialIntegration.java

```diff
@@ -145,14 +143,7 @@ public void integrate() {
   private void registerRepresentativeItem() {
     // also set the representative item
     if(material.getRepresentativeItem().isEmpty() && representativeItem != null && !representativeItem.isEmpty()) {
-      List<ItemStack> ore = OreDictionary.getOres(representativeItem, false);
```

Comment:
Wondering what the impact of existing mods is for that. Like XU2 that adds materials. Or IE.

Reply:
With the way I set it up, if they manually set the item there is no change since thats what the old code did with the ore names. If they use an oredict name will be the only changes, but they did not have control over that before anyways

Reply:
Ok, great